packages/app/src/cli/services/context/identifiers-extensions.test.ts

```diff
@@ -183,6 +185,7 @@ beforeAll(async () => {
 beforeEach(() => {
   vi.mocked(ensureAuthenticatedPartners).mockResolvedValue('token')
   vi.mocked(getUIExtensionsToMigrate).mockReturnValue([])
+  vi.mocked(partnersRequest).mockResolvedValue({app: {activeAppVersion: null}})
```

Comment:
hard to say from the code, what query to partners are you mocking here?

Reply:
TLDR: `fetchActiveAppVersion` (https://github.com/Shopify/cli/blob/app-access-module/packages/app/src/cli/services/dev/fetch.ts#L109)

More info:
1. `deployConfirmationPrompt` calls `buildInfoPrompt`
2. `buildInfoPrompt` calls `getInfoBreakdown`
3. `getInfoBreakdown` calls `fetchActiveAppVersion`
4. `fetchActiveAppVersion` calls `partnersRequest`

<img width="1624" alt="Screenshot 2023-11-13 at 05 56 36" src="https://github.com/Shopify/cli/assets/6130520/de906c98-94a1-4055-8653-33910d5e954d">